pyglmnet/pyglmnet.py

```diff
@@ -935,6 +935,36 @@ def predict_proba(self, X):
         yhat = np.asarray(yhat)
         return yhat
 
+    def predict_proba(self, X):
+        """Predict class probability for a binomial or probit distribution.
+
+        Parameters
+        ----------
+        X: array
+            Input data for prediction, of shape (n_samples, n_features)
+
+        Returns
+        -------
+        yhat: array
+            The predicted targets of shape (n_samples,).
+
+        Raises
+        ------
+        Works only for the binomial distribution.
+        Warn the output otherwise.
+
+        """
+        X = check_array(X, accept_sparse=False)
+        check_is_fitted(self, 'is_fitted_')
+
+        if self.distr in ['binomial', 'probit']:
+            return self._predict_proba(X)
+        else:
+            warnings.warn('This is only applicable for \
+                           the binomial distribution. \
+                           We returns predict as an output here.')
```

Comment:
scikit-learn does not like it if you raise an error for other distributions?

Reply:
Yeah, not quite sure what's the convention here. So, I did follow the use of `warnings` in `pyglmnet` instead.

Reply:
Let's go with this for now ...